### ISSUE TITLE
No weight decay?

### ISSUE THREAD

**Issue by lydials** [YU1ut/MixMatch-pytorch#22]
I notice that weight_decay is not set for Adam optimizer, see https://github.com/YU1ut/MixMatch-pytorch/blob/2df6c279a2106a9b87963c9350c70b7efbbb0e79/train.py#L115, but in the paper,  weight_decay is set to ~ 0.02.

**Comment by guixianjin** [comment]
Is this the weight decay?
[https://github.com/YU1ut/MixMatch-pytorch/blob/master/train.py#L388](url)

**Comment by lydials** [comment]
> Is this the weight decay?
> https://github.com/YU1ut/MixMatch-pytorch/blob/master/train.py#L388

thanks
